src/main/java/ca/unknown/bot/data_access/schedule_reminder/ScheduledReminderDAO.java

```diff
@@ -93,7 +114,7 @@ private Map<String, Schedule> load(String filename){
             // data which was written to it when saving the cache to a repo
             GsonBuilder builder = new GsonBuilder();
```

Comment:
Wouldn't Gsonbuilder be unnecessary if you are gonna be using new Gson();? Consider removing it.